test/SignatureValidator.t.sol

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+import { Test } from "forge-std/Test.sol";
+import { IERC1271Wallet } from "contracts/interfaces/IERC1271Wallet.sol";
+import { SignatureValidator } from "contracts/libraries/SignatureValidator.sol";
+import { MockERC1271Wallet } from "test/mocks/MockERC1271Wallet.sol";
+import { ECDSA } from "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
+
+contract SignatureValidatorTest is Test {
+    uint256 userPrivateKey = 1234;
+    uint256 walletAdminPrivateKey = 5678;
+    bytes32 digest = keccak256("EIP-712 data");
+    MockERC1271Wallet mockERC1271Wallet;
+
+    function setUp() public {
+        mockERC1271Wallet = new MockERC1271Wallet(vm.addr(walletAdminPrivateKey));
+    }
+
+    function testEIP712Signature() public {
```

Comment:
Also test `_signerAddress` being `address(0)` and `_signature` being empty bytes?